pkg/yurthub/kubernetes/meta/restmapper.go

```diff
@@ -193,7 +210,7 @@ func (rm *RESTMapperManager) UpdateKind(gvk schema.GroupVersionKind) error {
 // and delete the corresponding file in the disk (cache-crd-restmapper.conf), it should be used carefully.
 func (rm *RESTMapperManager) ResetRESTMapper() error {
 	rm.dynamicRESTMapper = make(map[schema.GroupVersionResource]schema.GroupVersionKind)
-	err := rm.storage.DeleteCollection(CacheDynamicRESTMapperKey)
+	err := rm.storage.DeleteFile(rm.cachedFilePath)
 	if err != nil {
 		return err
 	}
```

Comment:
how about modify Line 213~217 to `return rm.storage.DeleteFile(rm.cachedFilePath)`?

Reply:
fixed